### ISSUE TITLE
Have the API provide estimates for accessibility coefficients and theoretical accessibility coefficients 

### ISSUE THREAD

**Issue by agruss2** [jhpoelen/fb-osmose-bridge#93]
@jhpoelen Here is how accessibility coefficients (for focal functional groups) and theoretical accessibility coefficients (for background functional groups) should be estimated by the API. Hereafter, the term “accessibility coefficient” stands for “accessibility coefficient” or “theoretical accessibility coefficient”, since the two types of parameters should be estimated by the API exactly the same way. 
Let us assume that the API considers “Species 1” and “Species 2”. 
(1) The accessibility coefficient of Species 1 to Species 2 is determined as: accessibility coefficient = 0.8* coeff_1, where coeff_1 reflects the degree of overlap between functional groups in the water column. Coeff_1 can take the following values: (i) 1 (strong overlap in the water column); (ii) 0.5 (moderate overlap in the water column); or (iii) 0.1 (small overlap in the water column).
(2) In FishBase's ecology.csv, the API looks for the "Benthic", "Demersal" and "Pelagic" fields to determine whether: (i) Species 1 is benthic, demersal or pelagic; and (ii) Species 2 is benthic, demersal or pelagic. 
(3) In FishBase's estimatedepth.csv, the API looks for the "DepthMin" and "DepthMax" fields to determine whether the depth range (i.e., the interval [DepthMin DepthMax]) of Species 1 overlaps with the depth range of Species 2. 
(4) Finally: 
(i) If (Species 1 and Species 2 are both benthic OR are both demersal OR are both pelagic) AND (Species 1 and Species 2 have overlapping depth ranges), then coeff_1 = 1; 
(ii) If (Species 1 and Species 2 are not both benthic OR are not both demersal OR are not both pelagic) AND (Species 1 and Species 2 have overlapping depth ranges), then coeff_1 = 0.5;  
(iii) If (Species 1 and Species 2 are both benthic OR are both demersal OR are both pelagic) AND (Species 1 and Species 2 do not have overlapping depth ranges), then coeff_1 = 0.5; 
(ii) If (Species 1 and Species 2 are not both benthic OR are not both demersal OR are not both pelagic) AND (Species 1 and Species 2 do not have overlapping depth ranges), then coeff_1 = 0.1;  

Please let me know if you have any questions.

**Comment by jhpoelen** [comment]
First pass has been implemented as suggested. Note that mappings in ```fishbase-mapping.csv``` have been added to provide access to parameters and allow for setting of defaults.

Please verify.

**Comment by agruss2** [comment]
The present issue has been solved; therefore, I am closing it today.

**Comment by jhpoelen** [comment]
Is replaced by #172 .

**Comment by agruss2** [comment]
@jhpoelen Yes, the idea is to substitute the previous computations that the OSMOSE API were doing by new ones described in #172
